galleries/olof/index.html

```diff
@@ -0,0 +1,220 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Gallerix</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <center>
+            <h1>
+                Olofs Gallerix
+            </h1>
+        </center>
+    </header>
+
+    <div class="container">
+        <div class="aside">
+            <div>
+                <a href="#">idk</a>
+            </div>
+            <div>
+                <a href="#">idk2</a>
+            </div>
+        </div>
+        <div class="main">
+            <div class="main1" class="mainall">
+                <div class="box">
+                    <div class="date">
+                        <i>2020-10-11</i>
+                    </div>
+                    <img src="./images/boll.jpg" alt="" width="320" height="320">
+                    <div class="boll">
+                        <div class="like1">
+                            <div class="like2">
```

Comment:
Confusing class names. It seems like .like1 is a container/box and .like is a wrapper for the icon. Change names to something more clear.